src/test/java/org/broadinstitute/hellbender/tools/IndexFeatureFileIntegrationTest.java

```diff
@@ -42,17 +48,38 @@ public void testVCFIndex() {
         checkIndex(index, Arrays.asList("1", "2", "3", "4"));
     }
 
+    @Test(groups={"bucket"})
+    public void testVCFIndexOnCloud() throws IOException {
+        final File testFile = getTestFile("test_variants_for_index.vcf");
+        final String vcfOnGCS = BucketUtils.getTempFilePath(
+                getGCPTestStaging() +"testIndexOnCloud", ".vcf");
+        BucketUtils.copyFile(testFile.getAbsolutePath(), vcfOnGCS);
+
+        final String[] args = new String[] {
+                "IndexFeatureFile", "-I", vcfOnGCS
+        };
+
+        new Main().instanceMain(args);
```

Comment:
This is a bit unconventional, most tests just use `runCommandLine` there's nothing wrong with this though.